src/index.js

```diff
@@ -37,7 +38,15 @@ const allLevels = { error: 0, warn: 1, audit: 2, trace: 3, info: 4, perf: 5, ver
 const customLevelsArr = customLevels.split(/ *, */) // extra white space before/after the comma is ignored
 const ignoredLevels = customLevels ? Object.keys(allLevels).filter(key => !customLevelsArr.includes(key)) : []
 
-const customFormat = printf(({ level, message, timestamp }) => {
+const customFormat = printf(({ level, message, timestamp, context }) => {
```

Comment:
Unsure how to test this to stop coverage from failing.

Reply:
Wouldn't adding a test with the context resolve that?

Reply:
I was overthinking it of how to get at this function to mock it. Instead I wrote a test to spy of stdout to check that the context is formated properly.